ignite/cmd/plugin.go

```diff
@@ -293,24 +293,32 @@ func NewPlugin() *cobra.Command {
 	c.AddCommand(NewPluginList())
 	c.AddCommand(NewPluginUpdate())
 	c.AddCommand(NewPluginScaffold())
+	c.AddCommand(NewPluginDescribe())
 	return c
 }
 
 func NewPluginList() *cobra.Command {
-	return &cobra.Command{
+	lstCmd := &cobra.Command{
 		Use:   "list",
 		Short: "List declared plugins and status",
+		Long:  "Prints status and information of declared plugins",
 		RunE: func(cmd *cobra.Command, args []string) error {
 			printPlugins()
 			return nil
 		},
 	}
+
+	lstCmd.PersistentFlags().Bool("commands", false, "List information about plugin commands")
+	lstCmd.PersistentFlags().Bool("hooks", false, "List information about plugin hooks")
```

Comment:
I'm not sure those flags are still useful.

Reply:
removing them, thanks!